API/Controllers/TournamentsController.cs

```diff
@@ -75,21 +91,22 @@ public async Task<IActionResult> CreateAsync([FromBody] TournamentSubmissionDTO
     /// Get a tournament
     /// </summary>
     /// <param name="id">Tournament id</param>
-    /// <param name="unfiltered">If true, includes all match data, regardless of verification status.
-    /// Also includes all child navigations if true.
-    /// Default false (strictly verified data with limited navigation properties)
+    /// <param name="verified">If true, specifically includes verified match data. If false,
```

Comment:
```suggestion
    /// <param name="verified">
    /// If true, specifically includes verified match data. If false,
```
When the xml exceeds 1 line in length I would prefer to expand and keep the opening tag in it's own line